api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -230,13 +227,29 @@ private void maybeAlertOnCostThresholds(
    * have created. This is NOT live BigQuery data: it is only as recent as the last
    * checkFreeTierBillingUsage cron job, recorded as last_update_time in the DB.
    *
+   * <p>Note: return value may be null, to enable direct assignment to the nullable Profile field
+   *
    * @param user the user as represented in our database
    * @return the total USD amount spent in workspaces created by this user, represented as a double
    */
-  public Double getUserCachedFreeTierUsage(DbUser user) {
+  @Nullable
+  public Double getCachedFreeTierUsage(DbUser user) {
     return workspaceFreeTierUsageDao.totalCostByUser(user);
   }
 
+  /**
+   * Does this user have remaining free tier credits? Compare the user-specific free tier limit (may
+   * be the system default) to the amount they have used.
+   *
+   * @param user the user as represented in our database
```

Comment:
nit: can it be like user: the user as represented in our database to make it more readable

Reply:
I like that idea, but a space is the javadoc standard for `@param`